packages/manager/modules/billing/src/autoRenew/autorenew.controller.js

```diff
@@ -93,6 +94,10 @@ export default class AutorenewCtrl {
     }
   }
 
+  isAutorenew2016() {
+    return AUTORENEW_2016.includes(this.coreConfig.getUser().ovhSubsidiary);
```

Comment:
Well done, it was a mistake. UK => It's GB.
I don't think it's necessary to create a flipping feature because it's not really access to a function, it's different information depending on the subs.